internal/bminventory/inventory_test.go

```diff
@@ -4737,6 +4796,13 @@ func verifyApiError(responder middleware.Responder, expectedHttpStatus int32) {
 	ExpectWithOffset(1, conncreteError.StatusCode()).To(Equal(expectedHttpStatus))
 }
 
+func verifyApiErrorString(responder middleware.Responder, expectedHttpStatus int32, expectedSubstring string) {
+	ExpectWithOffset(1, responder).To(BeAssignableToTypeOf(common.NewApiError(expectedHttpStatus, nil)))
+	conncreteError := responder.(*common.ApiErrorResponse)
```

Comment:
typo?

Reply:
Right, a copied one :) Fixing

Reply:
Strange VSC spellchecker has nothing against the "alternative" spelling